### PR TITLE
ISSUE-915 Implement PostgresRateLimitingPlanUserRepository

### PR DESCRIPTION
**Pull request by quantranhong1999**
resolve https://github.com/linagora/tmail-backend/issues/915

### CODE REVIEW

**Review by chibenwa — COMMENTED** [review]
Read in diagonoal

**Review by Arsnael — APPROVED** [review]

**Review by vttranlina — APPROVED** [review]

File: tmail-backend/rate-limiter/rate-limiter-postgres/src/main/scala/com/linagora/tmail/rate/limiter/api/postgres/repository/PostgresRateLimitingPlanUserRepository.scala [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import javax.inject.Inject
+import org.apache.james.backends.postgres.utils.PostgresExecutor
+import org.apache.james.core.Username
+import org.reactivestreams.Publisher
+import reactor.core.publisher.Mono
+import reactor.core.scala.publisher.SMono
+
+class PostgresRateLimitingPlanUserRepository @Inject()(executorFactory: PostgresExecutor.Factory) extends RateLimitingPlanUserRepository {
+  private val daoWithoutRLS: PostgresRateLimitPlanUserDAO =
+    PostgresRateLimitPlanUserDAO(executorFactory.create())
```
Comment (hungphan227) [inline tmail-backend/rate-limiter/rate-limiter-postgres/src/main/scala/com/linagora/tmail/rate/limiter/api/postgres/repository/PostgresRateLimitingPlanUserRepository.scala:15]: use non-rls connection

Comment (quantranhong1999) [reply tmail-backend/rate-limiter/rate-limiter-postgres/src/main/scala/com/linagora/tmail/rate/limiter/api/postgres/repository/PostgresRateLimitingPlanUserRepository.scala:15]: It is needed. Please read the code more to know why.

Comment (quantranhong1999) [reply tmail-backend/rate-limiter/rate-limiter-postgres/src/main/scala/com/linagora/tmail/rate/limiter/api/postgres/repository/PostgresRateLimitingPlanUserRepository.scala]: Very good catch indeed. Thanks @hungphan227

**Review by hungphan227 — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Pull request merged**